src/server/admin/decommissions/controllers/confirm-decommission.js

```diff
@@ -32,14 +26,29 @@ const confirmDecommissionController = {
       return h.redirect(request.routeLookup('admin/decommissions/start'))
     }
 
-    const validationResult = decommissionValidation(repositoryNames).validate(
-      { repositoryName },
-      { abortEarly: false }
-    )
+    const entityToDecommission = await fetchEntity(repositoryName)
 
-    if (validationResult?.error) {
-      const errorDetails = buildErrorDetails(validationResult.error.details)
+    if (!entityToDecommission) {
+      const errorDetails = {
+        repositoryName: {
+          message: 'Choose an entry'
+        }
+      }
+      request.yar.flash(sessionNames.validationFailure, {
+        formValues: { repositoryName },
+        formErrors: errorDetails
+      })
+
+      return h.redirect(request.routeLookup('admin/decommissions/start'))
+    }
 
+    if (isPostgresDeleteProtectionEnabled(entityToDecommission)) {
+      const errorDetails = {
+        repositoryName: {
+          message:
+            'Postgres delete protection must be disabled before decommissioning.'
```

Comment:
Bit of an odd UX this error appearing as an inline form error. Feels like its a `globalValidationFailures` error?